notebooks/train.py

```
@@ -0,0 +1,182 @@
+import pandas as pd
+from autogluon.timeseries import TimeSeriesDataFrame, TimeSeriesPredictor
+import mlflow
+import os
+import logging
+import random
+
+# Ensure /tmp directory exists
+os.makedirs('/tmp', exist_ok=True)
```

Ruff (0.12.2)

Probable insecure usage of temporary file or directory: "/tmp"

(S108)